pennylane/operation.py

```diff
@@ -1206,15 +1206,36 @@ def expand(self):
         return tape
 
     def __add__(self, other):
-        r"""The addition operation between Operator objects."""
-        if isinstance(other, numbers.Number) and other == 0:
-            return self
+        """The addition operation of Operator-Operator objects and Operator-scalar."""
+        if isinstance(other, numbers.Number):
+            if other == 0:
+                return self
+            return qml.ops.Sum(  # pylint: disable=no-member
+                self,
+                qml.ops.SProd(  # pylint: disable=no-member
+                    scalar=other, base=qml.Identity(wires=self.wires)
+                ),
```

Comment:
The linter puts it automatically. Since I need to add the `# pylint: disable=no-member` comment for the `Sum` and `SProd` classes, this is the only way of doing it without exceeding the maximum line length.

Reply:
There is a similar example where the comma is used:
https://github.com/PennyLaneAI/pennylane/blob/14485da8eae88ac749f2bc2e1a8e7b5678feccc6/pennylane/operation.py#L545